workbench-core/datasets/src/dataSetService.ts

```diff
@@ -146,7 +162,22 @@ export class DataSetService {
    * @returns - the DataSet object associated with that DataSet.
    */
   public async getDataSet(dataSetId: string): Promise<DataSet> {
-    return await this._dbProvider.getDataSetMetadata(dataSetId);
+    const metadata: Metadata = {
+      actor: { id: '-', roles: [] },
```

Comment:
could this be more specific?

Reply:
Not without a late game breaking change to a method we did not claim to be updating.